common/lib/xmodule/xmodule/modulestore/split_mongo/mongo_connection.py

```diff
@@ -438,6 +438,7 @@ def get_course_index(self, key, ignore_case=False):
         """
         Get the course_index from the persistence mechanism whose id is the given key
         """
+        query = {}
```

Comment:
Is this line still needed? `query` will be set by one of the following `if` or `else` statements.